cmd/tezosconnect_test.go

```diff
@@ -2,72 +2,42 @@ package cmd
 
 import (
 	"context"
-	"os"
 	"testing"
 	"time"
 
-	"github.com/hyperledger/firefly-common/pkg/config"
-	"github.com/hyperledger/firefly-tezosconnect/internal/tezos"
 	"github.com/stretchr/testify/assert"
 )
 
 func TestRun(t *testing.T) {
 	testCases := []struct {
-		name        string
-		errMsg      string
-		initFunc    func()
-		cleanupFunc func()
+		name    string
+		errMsg  string
+		cfgFile string
 	}{
 		{
-			name: "success",
-			initFunc: func() {
-				f, err := os.Create("firefly.tezosconnect")
-				assert.NoError(t, err)
-				err = f.Close()
-				assert.NoError(t, err)
-
-				dir, err := os.MkdirTemp("", "ldb_*")
-				assert.NoError(t, err)
-				config.Set("persistence.leveldb.path", dir)
-			},
-			cleanupFunc: func() {
-				err := os.Remove("firefly.tezosconnect")
-				assert.NoError(t, err)
-			},
+			name:    "success",
+			cfgFile: "../test/firefly.tezosconnect.yaml",
 		},
 		{
-			name:        "error on config not found",
-			initFunc:    func() {},
-			cleanupFunc: func() {},
-			errMsg:      "FF00101: Failed to read config: Config File \"firefly.tezosconnect\" Not Found",
+			name:    "error on config not found",
+			cfgFile: "../test/missing.firefly.tezosconnect.yaml",
+			errMsg:  "FF00101: Failed to read config: open ../test/missing.firefly.tezosconnect.yaml: no such file or directory",
 		},
 		{
-			name: "error on NewTezosConnector",
-			initFunc: func() {
-				f, err := os.Create("firefly.tezosconnect")
-				assert.NoError(t, err)
-				err = f.Close()
-				assert.NoError(t, err)
-
-				connectorConfig.Set(tezos.TxCacheSize, "-1")
-			},
-			cleanupFunc: func() {
-				err := os.Remove("firefly.tezosconnect")
-				assert.NoError(t, err)
-			},
-			errMsg: "FF23040: Failed to initialize transaction cache: Must provide a positive size",
+			name:    "error on NewTezosConnector",
+			cfgFile: "../test/firefly.tezosconnect-without-connector.yaml",
+			errMsg:  "FF23051: Blockchain RPC node URL must be set",
 		},
 	}
 
 	for _, tc := range testCases {
 		t.Run(tc.name, func(t *testing.T) {
-			tc.initFunc()
+			InitConfig()
```

Comment:
Do we need to init the config for each testcase? Or it's enough to do it just once and we can move it out of the loop